integreat_cms/cms/views/translations/translations_management_view.py

```diff
@@ -25,6 +29,55 @@ class TranslationsManagementView(TemplateView):
     #: The context dict passed to the template (see :class:`~django.views.generic.base.ContextMixin`)
     extra_context = {"current_menu_item": "translations_management"}
 
+    def get_context_data(self, **kwargs):
+        r"""
+        Extend context by word counts
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :return: The context dictionary
+        :rtype: dict
+        """
+
+        region = self.request.region
+        content_types = [Event, POI, Page]
+
+        word_count = {}
+
+        for content_type in content_types:
+            content_name = content_type._meta.verbose_name_plural.title()
+            word_count[content_name] = Counter()
+            for status, name in CHOICES:
+                word_count[content_name][status] = 0
+
+            contents = (
+                content_type.objects.filter(region=region, archived=False)
+                if not content_type == Page
+                else region.get_pages()
+            )
+            for content in contents:
+                if latest_version := content.get_translation(
+                    region.default_language.slug
+                ):
+                    word_count[content_name][latest_version.status] += len(
+                        latest_version.content.split()
+                    )
```

Comment:
I'm not sure this is always what we want - e.g. if the latest version is an autosave, I guess we still want to count the words in the latest public version? (see my comment in https://github.com/digitalfabrik/integreat-cms/issues/1845#issuecomment-1616708893, waiting for further clarification...)